product_docs/docs/edbcloud/beta/getting_started/01_understanding_qotas_in_azure.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Understanding quotas in Azure
+Redirects:
+ - 01_check_resource_limits/e3
+---
+
+The following sections help you plan for quotas (service limits) and Azure Service Provider registrations required by your cluster.
+
+[Managing service limits and constraints in Azure](01a_managing_quotas) describes how to check that your clusters meet these quotas and how to raise your resource limits, if needed.
+
+### IP Address quota
+
+You need to increase the limit of `Public IP Addresses - Basic` and `Public IP Addresses - Standard` for the regions where you plan to deploy your clusters with the total number of clusters you plan to use. To understand the requirements, and recommended quotas, see the following sections.
```

Comment:
The "managing service limits" page is the thing that should describe what the customer needs to do, right? This page should describe why BA requires changes to these things, e.g., "For every public-network cluster, BA deploys a dedicated IP address", et cetera.

Reply:
added this under recommended limits:

"EDB recommends you to increase the Public IP addresses limits for the **Basic** and **Standard** type to the number of clusters you plan to deploy in a given region."